docs/content/en/docs/pipeline-stages/builders/docker.md

```diff
@@ -67,15 +67,37 @@ These credentials are configured in the `cluster` section with the following opt
 
 {{< schema root="ClusterDetails" >}}
 
-To set up the credentials for Kaniko refer to the [kaniko docs](https://github.com/GoogleContainerTools/kaniko#kubernetes-secret) (**Note**: Rename the downloaded JSON key to *kaniko-secret* without appending *.json*).
-Alternatively, the path to a credentials file can be set with the `pullSecret` option:
+### Configure Kaniko Credentials 
+
+To set up the credentials for Kaniko refer to the [kaniko docs](https://github.com/GoogleContainerTools/kaniko#kubernetes-secret)
+
+(**Note**: Rename the downloaded JSON key to *kaniko-secret* without appending *.json*).
+
+Alternatively, the path to credentials file can be set with the `pullSecretPath` option:
```

Comment:
What is this alternative to?  The `(**Note**: …` or the `To set up the credentials …`?

Reply:
The former (i.e., `**Note**`).

Recently `pullSecretPath` was converted to serve as a dual-purpose property (#4381) to address the issue that Skaffold uses the hard-coded secret key name `kaniko-secret`. So, instead of renaming the JSON key file to `kaniko-secret` (without `.json`), you can now say, e.g., `pullSecretPath: my-key.json` if you created your secret with
```
kubectl create secret generic kaniko-secret --from-file=.../my-key.json
```
Another alternative not documented here is to explicitly specify the key name with
```
kubectl create secret generic kaniko-secret --from-file=kaniko-secret=.../my-key.json
```